wiki/Installation/en.md

```diff
@@ -20,6 +22,19 @@ This page will roughly tell you how to get osu! working on your Windows device.
 5.  osu! will prompt you to sign in or register.
     -   If you have an account already, sign in.
     -   If you need to register, see [Registration](/wiki/Registration), then sign in.
+    
+### MacOS
+
+- Since the MacOS version of osu! is unofficial, the game may not fuction properly, or even boot at all. 
+
+1.  Go to the unofficial [download page](https://osu.ppy.sh/community/forums/topics/682197) and download the installer.
+2.  Locate the installer, right click, and select open.
+    - Double left clicking on the file may not open it, since MacOS may recognize the app as developed by an unidentified devloper. Right clicking and selecting open will solve this problem.
```

Comment:
```suggestion
    -   Double left clicking on the file may not open it, since MacOS may recognize the app as developed by an unidentified developer. Right clicking and selecting open will solve this problem.
```

Reply:
was the space at the front intentional

Reply:
Yes, it would keep the spacing consistent with the numbered list.